package.json

```diff
@@ -30,8 +30,6 @@
     "releaseNext": "release-it --npm.tag=next --preRelease=rc",
     "example": "yarn --cwd example",
     "bootstrap": "yarn example && yarn && yarn example pods",
-    "fabricexample": "yarn --cwd fabricexample",
-    "bootstrap-fabric": "yarn fabricexample && yarn && yarn fabricexample pods",
```

Comment:
How about leaving this `bootstrap-fabric` script and making it actually bootstrap fabric in the `example` folder?